support/Procfile

```diff
@@ -1,4 +1,3 @@
-web: support/builder_web.sh
```

Comment:
Do we need to remove this? I kind of like it being there for those who'd like a UI but don't want to fuss with having to set up to run one locally. 

If we do want to remove it, I believe the builder_web script was included with the PR that introduced this line, so perhaps we should remove that as well.